Dockerfile

```
@@ -14,7 +14,7 @@ ARG PGDATA
 ARG LC_ALL
 ARG LANG
 
-ENV ETCDVERSION=3.3.13 CONFDVERSION=0.16.0
+ENV ETCDVERSION=3.6.4 CONFDVERSION=0.16.0
```

I choose this specific version of Etcd because it has minimal size of binaries.
```suggestion
ENV ETCDVERSION=3.3.13 CONFDVERSION=0.16.0
```